apps/onboarding/src/config/session.config.ts

```diff
@@ -0,0 +1,14 @@
+import { registerAs } from '@nestjs/config'
+
+export default registerAs('session', () => ({
```

Comment:
let's register this as `database` and call it database.config - it's not specifically for the session module.